src/index.ts

```diff
@@ -352,20 +352,33 @@ const mergeDeep = (...objects: readonly any[]): any => {
 const glitchElement = (element: HTMLElement, layers: LayerDefinition[], options: PowerGlitchOptions): { container: HTMLDivElement, startGlitch: () => void, stopGlitch: () => void } => {
     const alreadyGlitched = !! element.dataset.glitched;
 
-    // Container
+    /**
+     * Depending on the element state (whether it was glitched before or not, current element display attributes)
+     */
     let container: HTMLDivElement;
+    let layersContainer: HTMLDivElement;
```

Comment:
Had to introduce another container to allow for inlining the entire glitch container